utils/docker-ce/files/etc/config/dockerd

```diff
@@ -0,0 +1,7 @@
+
+config globals 'globals'
+	option alt_config_file "/etc/daemon.json"
```

Comment:
This might get reworked depending on how you want to address the previous comments, so FYI, this looks like it should have been
```suggestion
	option alt_config_file "/etc/docker/daemon.json"
```

Reply:
I think we should do the link.
I've seen this before in openWrt.

- https://github.com/openwrt/packages/blob/master/net/keepalived/files/keepalived.init#L500
- https://github.com/openwrt/packages/blob/master/net/stunnel/files/stunnel.init#L342